calico/felix/test/test_ipsets.py

```diff
@@ -518,16 +539,46 @@ def setUp(self):
     @patch("calico.felix.futils.check_call", autospec=True)
     def test_replace_members(self, m_check_call):
         self.ipset.replace_members(set(["10.0.0.1"]))
-        m_check_call.assert_called_once_with(
-            ["ipset", "restore"],
-            input_str='create foo hash:ip family inet --exist\n'
-                      'create foo-tmp hash:ip family inet --exist\n'
-                      'flush foo-tmp\n'
-                      'add foo-tmp 10.0.0.1\n'
-                      'swap foo foo-tmp\n'
-                      'destroy foo-tmp\n'
-                      'COMMIT\n'
-        )
+        exp_calls = [
+            call(["ipset", "destroy", "foo-tmp"]),
```

Comment:
Note to self: figure out why we've gained these destroy calls.  Why are they in a separate commit?  Do we still need the flush?